### PR TITLE
Lydia/rs 16 error tracking with sentry

### PR DESCRIPTION
**Pull request by LydiaKoil**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -27,6 +27,10 @@ def after_sign_in_path_for(_resource)
 
   private
 
+  def set_sentry_user
```
Comment (danielpaul) [inline app/controllers/application_controller.rb:30]: this method isn't getting called from anywhere?

Comment (danielpaul) [reply app/controllers/application_controller.rb:30]: https://github.com/IdentitySquare/IntergalacticFootball/blob/c99fabd4938735beb49390ffc2c4e5cafaaab61a/app/controllers/application_controller.rb#L11

File: app/views/layouts/components/_head.html.haml [hunk 2]
```diff
@@ -9,4 +9,16 @@
...
+= javascript_importmap_tags
+
+
+- if Rails.env.production? && ENABLE_SENTRY == true
+    -# JS Error Tracking 
+    %script{crossorigin: "anonymous", integrity: "sha384-davt05dZyy9wSVcYU+nxmOaNugUmN7v0WYFUlbE6d6kVDbYxqWKzJXY7ZM04YbJf", src: "https://browser.sentry-cdn.com/7.8.0/bundle.tracing.min.js"}
+
+    :javascript
+      Sentry.init({
+        dsn: SENTRY_DSN_JS,
```
[the remark this answers is not in the document]
Comment (danielpaul) [reply app/views/layouts/components/_head.html.haml:21]: the value also need to be inside `""` and not just printed when its displayed. the output should be the same as what you would expect if the value was hard coded here. have a look at the output HTML to confirm.

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: app/views/layouts/components/_head.html.haml (hunk 2 above)
[the remark this answers is not in the document]
Comment (LydiaKoil) [reply app/views/layouts/components/_head.html.haml:21]: image

**Review by danielpaul — APPROVED** [review]

**Pull request merged**